Octokit/Models/Request/SearchCodeRequest.cs

```diff
@@ -18,6 +18,12 @@ public class SearchCodeRequest : BaseSearchRequest
     {
         public SearchCodeRequest(string term) : base(term) { }
 
+        public SearchCodeRequest(string term, string repo)
```

Comment:
This is a little confusing. I assume `repo` here should actually be the repo name and repo owner. For example, `octokit/octokit.net` as opposed to just `octokit.net`.

I think we should probably use the pattern we use elsewhere and have separate parameters for the owner and the name and then concatenate it ourselves.

```csharp
public SearchCodeRequest(string term, string owner, string name)...
```

/cc @shiftkey for thoughts in case I'm missing something.